### ISSUE TITLE
build docs failed

### ISSUE THREAD

**Issue by ipdae** [planetarium/libplanet#595]
```
PS C:\Users\qoora\projects\libplanet\Docs> .\build.ps1
& : 'C:\Users\qoora\projects\libplanet\Docs\docfx\docfx.exe' 용어가 cmdlet, 함수, 스크립트 파일 또는 실행할 수 있는 프
로그램 이름으로 인식되지 않습니다. 이름이 정확한지 확인하고 경로가 포함된 경우 경로가 올바른지 검증한 다음 다시 시도하
십시오.
위치 C:\Users\qoora\projects\libplanet\Docs\build.ps1:92 문자:7
+     & "$BaseDir\docfx\docfx.exe" "$BaseDir/docfx.json" @args
+       ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\qoora\...docfx\docfx.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException```

**Comment by longfin** [comment]
It seems that there is no `docfx.exe`. (but we're [installing it on `build.ps1`](https://github.com/planetarium/libplanet/blob/9496f04323fe94bc091cd63a58b615eeb4cfb99b/Docs/build.ps1#L50-L61).)

@ipdae Could you provide full command line logs?

**Comment by ipdae** [comment]
@longfin i don`t have any no more logs.

**Comment by longfin** [comment]
@ipdae Do you have `<project_root>\Docs\docfx` directory? if so, please retry after deleting it.

**Comment by ipdae** [comment]
```
PS C:\Users\qoora\projects\libplanet\Docs> ls


    디렉터리: C:\Users\qoora\projects\libplanet\Docs


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----     2019-03-07   오후 3:05                api
d-----     2019-03-07   오후 3:05                articles
d-----     2019-10-21   오후 1:22                docfx
d-----     2019-03-07   오후 3:05                images
-a----     2019-03-07   오후 3:05            171 .gitignore
-a----     2019-10-21   오후 1:17           3213 build.ps1
-a----     2019-03-07   오후 3:05             40 CONTRIBUTING.md
-a----     2019-05-07  오전 11:55           1678 docfx.json
-a----     2019-03-07   오후 3:05             34 index.md
-a----     2019-10-21  오후 12:13           3378 publish.sh
-a----     2019-03-07   오후 3:05            238 toc.yml


PS C:\Users\qoora\projects\libplanet\Docs> rmdir docfx
PS C:\Users\qoora\projects\libplanet\Docs> ls


    디렉터리: C:\Users\qoora\projects\libplanet\Docs


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----     2019-03-07   오후 3:05                api
d-----     2019-03-07   오후 3:05                articles
d-----     2019-03-07   오후 3:05                images
-a----     2019-03-07   오후 3:05            171 .gitignore
-a----     2019-10-21   오후 1:17           3213 build.ps1
-a----     2019-03-07   오후 3:05             40 CONTRIBUTING.md
-a----     2019-05-07  오전 11:55           1678 docfx.json
-a----     2019-03-07   오후 3:05             34 index.md
-a----     2019-10-21  오후 12:13           3378 publish.sh
-a----     2019-03-07   오후 3:05            238 toc.yml


PS C:\Users\qoora\projects\libplanet\Docs> .\build.ps1
Invoke-WebRequest : 매개 변수 이름 'L'과(와) 일치하는 매개 변수를 찾을 수 없습니다.
위치 C:\Users\qoora\projects\libplanet\Docs\build.ps1:42 문자:10
+     curl -L -o "$To" "$From"
+          ~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-WebRequest], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestCommand



    디렉터리: C:\Users\qoora\projects\libplanet\Docs


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----     2019-10-21   오후 1:22                docfx
"2"개의 인수가 있는 "ExtractToDirectory"을(를) 호출하는 동안 예외가 발생했습니다. "'C:\Users\qoora\projects\libplanet\D
ocs\docfx.zip' 파일을 찾을 수 없습니다."
위치 C:\Users\qoora\projects\libplanet\Docs\build.ps1:56 문자:3
+   [System.IO.Compression.ZipFile]::ExtractToDirectory(
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : FileNotFoundException

Remove-Item : 'C:\Users\qoora\projects\libplanet\Docs\docfx.zip' 경로는 존재하지 않으므로 찾을 수 없습니다.
위치 C:\Users\qoora\projects\libplanet\Docs\build.ps1:60 문자:3
+   Remove-Item "$BaseDir/docfx.zip"
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\qoora\...\Docs\docfx.zip:String) [Remove-Item], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand

& : 'C:\Users\qoora\projects\libplanet\Docs\docfx\docfx.exe' 용어가 cmdlet, 함수, 스크립트 파일 또는 실행할 수 있는 프
로그램 이름으로 인식되지 않습니다. 이름이 정확한지 확인하고 경로가 포함된 경우 경로가 올바른지 검증한 다음 다시 시도하
십시오.
위치 C:\Users\qoora\projects\libplanet\Docs\build.ps1:92 문자:7
+     & "$BaseDir\docfx\docfx.exe" "$BaseDir/docfx.json" @args
+       ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\qoora\...docfx\docfx.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

delete `docfx`. but, same problem.

**Comment by longfin** [comment]
```
Invoke-WebRequest : 매개 변수 이름 'L'과(와) 일치하는 매개 변수를 찾을 수 없습니다.
위치 C:\Users\qoora\projects\libplanet\Docs\build.ps1:42 문자:10
+     curl -L -o "$To" "$From"
+          ~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-WebRequest], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
```

It seems installed `curl` doesn't know `-L` option. 

@ipdae Could you check what `curl` is installed using `Get-Command curl`?

**Comment by dahlia** [comment]
Fixed by the patch #596.
